plugins/modules/mount.py

```diff
@@ -105,6 +105,14 @@
         the original file back if you somehow clobbered it incorrectly.
     type: bool
     default: no
+  umask:
+    description:
+      - The umask to set before creating new directory(ies) for the mount point.
+        If the mount point already exists, this parameter is not used.
+      - Note that after running this task and the device being successfully mounted,
+        the mode of the original directory will be hidden by the target device.
+    type: raw
```

Comment:
I would change this to a 'strict string' to avoid all the 'octal/int' issues as we had to go through with mode in other modules.